deploy/nv-ipam.yaml

```diff
@@ -52,6 +52,7 @@ spec:
         app: nv-ipam-node
         name: nv-ipam-node
     spec:
+      hostNetwork: true
```

Comment:
@moshe010 do you want to submit PR to add host network to node ?
so that it dont depends on CNI